src/main/java/mate/academy/internetshop/controllers/AddProductToShoppingCartController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+public class AddProductToShoppingCartController extends HttpServlet {
+    private static Long USER_ID = 1L;
+
+    private static final Injector injector = Injector.getInstance("mate.academy");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
```
пройдися пліз по всіх контролерах та виправ аналогічні помилки